src/woo_publications/publications/tests/test_publications_api.py

```diff
@@ -1342,12 +1342,12 @@ def test_create_publication(self):
                 "verkorteTitel": "one",
                 "omschrijving": "Lorem ipsum dolor sit amet, "
                 "consectetur adipiscing elit.",
-                # These values will be overwritten
-                "bronBewaartermijn": "THIS VALUE WON'T BE USED",
-                "selectiecategorie": "THIS VALUE WON'T BE USED",
+                # These values will not be overwritten because the status is concept
+                "bronBewaartermijn": "THIS VALUE WILL BE USED",
+                "selectiecategorie": "THIS VALUE WILL BE USED",
                 "archiefnominatie": ArchiveNominationChoices.destroy,
```

Comment:
Do we have a similar test case to the original for creation of a published publication?

The original behaviour is still valid for published publications, so just updating the test like that creates the impression that the submitted data always overrules the computed value, but that's not the case. Please add a test for publication creation of a published publication with these fields set and assert that the automatically derived values are used.